EPS: 20
ANGLE_EPS: 5


# Test params
MARKER_SIZE: 0.02
ROBOT_SIZE: 0.05
ROBOT_H: 0.03
WHEEL_SIZE: 0.03

CONNECTION_DISTANCE: 0.01


## BIG MARS PARAMETRS

# MARKER_SIZE: 0.047
# ROBOT_SIZE: 0.26
# ROBOT_H: 0.14
# WHEEL_SIZE: 0.14
#
# CONNECTION_DISTANCE: 0.02
